### ISSUE TITLE
Loss can't be below 1.

### ISSUE THREAD

**Issue by BlueAnthony** [AlexeyAB/darknet#829]
The loss can not decrease under 1, it will stop and jitter around some number, like 5~6 or 12~13.
I use yolov3.weight as pretrained.
![image](https://user-images.githubusercontent.com/17959032/40092523-46d96fce-58f0-11e8-8822-6faab9d42786.png)

**Comment by AlexeyAB** [comment]
Use these params: 
https://github.com/AlexeyAB/darknet/blob/e29fcb703f8d936e17507bf78043a8b8bc6279b0/cfg/yolov3.cfg#L18-L23

And train about 2000 iterations.
If it doesn't help, then something wrong with classes number or with your dataset. Check it using this software: https://github.com/AlexeyAB/Yolo_mark

* Do you use `random=1` in cfg-file?
* Do you use the latest version of this GitHub repository? https://github.com/AlexeyAB/darknet

**Comment by BlueAnthony** [comment]
@AlexeyAB When I change to use "AlexeyAB/darknet",  "./darknet detector train cfg/kitti.data cfg/yolov3-kitti.cfg model/yolov3.weights" will directly save original model without training
![image](https://user-images.githubusercontent.com/17959032/40156287-2329ede8-59cb-11e8-80c6-ac119ef5974b.png)

**Comment by panda9095** [comment]
Your learning rate is about 1e-8 which is too small. Try using the option of -clear, then the iteration will restart from 0, if you'd like to use yolov3.weights as pre-trained weights.

**Comment by BlueAnthony** [comment]
@panda9095 Thank you for your response.
Could you tell me more detail about "-clear"? When will "-clear" restart?

**Comment by panda9095** [comment]
@BlueAnthony ./darknet detector train cfg/kitti.data cfg/yolov3-kitti.cfg model/yolov3.weights **-clear**

By doing so, the step number will start from 0 instead of 500200. Then you can use @AlexeyAB 's parameters for training.

**Comment by BlueAnthony** [comment]
@panda9095 Really thank you for your helping!

**Comment by AlexeyAB** [comment]
@BlueAnthony 
Properly commands for training:

1. `./darknet detector train cfg/kitti.data cfg/yolov3-kitti.cfg model/yolov3.weights -clear`

2. `./darknet detector train cfg/kitti.data cfg/yolov3-kitti.cfg darknet53.conv.74`

3. `./darknet detector train cfg/kitti.data cfg/yolov3-kitti.cfg yolov3.conv.105`
   Pre-trained file yolov3.conv.105 you can get by using this command: 
    `./darknet partial cfg/yolov3.cfg yolov3.weights yolov3.conv.105 105`
  
  https://github.com/AlexeyAB/darknet/blob/fb9fcfb3ba0057b1196d1b95fd53cdb0b874b1c3/build/darknet/x64/partial.cmd#L21

**Comment by hahazhky** [comment]
@AlexeyAB Hi, AlexeyAB, when I finetue the network, if I want to fix some layers weights, what should I do, what about setting corresponding layers' learning_rate=0(to change the learning rate scale), or there is something better to do

**Comment by AbhimanyuAryan** [comment]
@AlexeyAB does this mean that I can further train my last trained model(on my dataset)....with new data?
